clients/python/client/osparc/models.py

```diff
@@ -22,12 +22,9 @@
     Meta,
     Profile,
     ProfileUpdate,
-    Solver,
-    TaskStates,
-    UserRoleEnum,
-    UsersGroup,
-    ValidationError,
 )
+from osparc_client.models import RunningState as TaskStates
+from osparc_client.models import Solver, UserRoleEnum, UsersGroup, ValidationError
 
 __all__ = [
```

Comment:
MINOR: use tuple instead of list for all, i.e.
```
__all__ : tuple[str, ...] = ( "BodyUploadFileV0FilesContentPut",  )
```

Reply:
done

Reply:
still a list?

Reply:
Sorry, I managed to fix it in `__init__.py` but not in `models.py` and `api.py`. That's done now